packages/components/src/components/CodeEditor.tsx

```diff
@@ -200,6 +200,7 @@ export const CodeEditor = forwardRef<CodeEditorHandle, CodeEditorProps>(
         effects: compTheme.reconfigure(
           EditorView.theme({
             "&": {
+              overflow: "auto",
```

Comment:
Is this necessary in this version? I assume it was necessary in Cab's version, but in this PR all panels already have `overflow: auto` set, so I'm not sure.

Reply:
Good point, removed.